ibm_catalog.json

```diff
@@ -182,6 +223,34 @@
                   }
                 }
               },
+              {
+                "key": "cspm_enabled"
+              },
+              {
+                "key": "app_config_crn"
```

Comment:
It might be a better user experience to mark this as required (so it shows in the required tab) since the default value of `cspm_enabled` is true. Otherwise I think consumers will end of trying a terraform plan, and it failing because they never added a value for `app_config_crn`.

If we do this, lets ensure to update the terraform variable description to say that the value can be set to `null` if `cspm_enabled` is set to false